pkg/providers/property/provider.go

```diff
@@ -84,6 +84,7 @@ func Provider() *schema.Provider {
 			"akamai_properties":              dataSourceAkamaiProperties(),
 			"akamai_property_products":       dataSourceAkamaiPropertyProducts(),
 			"akamai_property_hostnames":      dataSourceAkamaiPropertyHostnames(),
+			"akamai_search":                  dataSourceSearch(),
```

Comment:
```suggestion
			"akamai_property_search":                  dataSourcePropertySearch(),
```

Reply:
since this, I will also change all the file names.